jobs/pacman-cloud-discovery/src/main/java/com/tmobile/cso/pacman/inventory/file/AssetFileGenerator.java

```diff
@@ -85,6 +85,7 @@ public void generateFiles(List<Map<String,String>> accounts,String skipRegions,S
 			FileManager.initialise(filePath);
 			ErrorManageUtil.initialise();
 		} catch (IOException e1) {
+			ErrorManageUtil.jobStatus = "failed";
```

Comment:
same here - wrong usage of utility class

Reply:
moved the jobStatus attribute to a different class.